app/MindWork AI Studio/Components/ProfileSelection.razor

```diff
@@ -1,6 +1,6 @@
 @inherits MSGComponentBase
 <MudTooltip Text="@T("You can switch between your profiles here")" Placement="Placement.Top">
-    <MudMenu TransformOrigin="@Origin.BottomLeft" AnchorOrigin="Origin.TopLeft" StartIcon="@Icons.Material.Filled.Person4" EndIcon="@Icons.Material.Filled.KeyboardArrowDown" Label="@this.CurrentProfile.Name" Variant="Variant.Filled" Color="Color.Default" Class="@this.MarginClass">
+    <MudMenu TransformOrigin="@Origin.BottomLeft" AnchorOrigin="Origin.TopLeft" StartIcon="@Icons.Material.Filled.Person4" EndIcon="@Icons.Material.Filled.KeyboardArrowDown" Label="@this.CurrentProfile.Name" Variant="Variant.Filled" Color="Color.Default" Class="@this.MarginClass" disabled="@this.Disabled">
```

Comment:
It should be `Disabled` instead of `disabled`. Lowercase usually refers to the HTML5 attribute, while the capitalized version typically refers to Blazor or MudBlazor.